### ISSUE TITLE
Add `textResourceBindings` to `Address` component

### ISSUE THREAD

**Issue by nkylstad** [Altinn/app-frontend-react#36]
# [See comment below for an up-to-date description](https://github.com/Altinn/app-frontend-react/issues/36#issuecomment-1733649690)

## Description
We have several cases of components that do not have overall labels. F.ex. the address component - this has labels for each of the input fields, but no "title" text for the component as a whole (although app developers can set this in studio, it's not shown anywhere).
The current solution has been to add a title/paragraph component for any extra title texts. This works visually, but is not compatible with screen readers in form mode, so any such texts would be skipped. We need to find a way to add a title text to any component/group, and connect it to the form component/group in some way. 

## Screenshots
> Screenshots or links to Figma (make sure your sketch is public)

## Considerations
> Describe input (beyond tasks) on how the user story should be solved can be put here.
Some options:
- Have title texts with ALL form components, incl. f.ex. address component and group.
- Allow app developers to connect text components (title/paragraph) to specific form components
- Have app developers code this manually in the text (title/paragraph component) using `<label>`-tags
  - This is already a possibility, but we would need to check that it works.

## Acceptance criteria
> Describe criteria here (i.e. What is allowed/not allowed (negative tesing), validations, error messages and warnings etc.)

## Specification tasks

- [ ] Development tasks are defined
- [ ] Test design / decide test need

## Development tasks
> Add tasks here

## Definition of done
Verify that this issue meets [DoD](https://confluence.brreg.no/display/T3KP/Definition+of+Done#DefinitionofDone-DoD%E2%80%93utvikling) (Only for project members) before closing.

- [ ] Documentation is updated (if relevant)
  - [ ] Technical documentation (docs.altinn.studio)
  - [ ] User documentation (altinn.github.io/docs)
- [ ] QA
- [ ] Manual test is complete (if relevant)
- [ ] Automated test is implemented (if relevant)
- [ ] All tasks in this userstory are closed (i.e. remaining tasks are moved to other user stories or marked obsolete)

**Comment by olemartinorg** [comment]
I think this might be a bit outdated now. We should not force every component to support a title or paragraph/description binding, as it will not always make sense for them to have that, and the Address component is a great example of that.

For the address component specifically, in this issue we should make sure to expose bindings for the individual form fields inside that component. I.e. overriding the title (at least) for:
- Address
- Zip code
- Post place
- C/O
- House number

**Comment by hflateland** [comment]
Any chance of progress on this one? Even just changing the label from "Gateadresse" to "Adresse" would help, since in it current form it's not possible to use the address component correctly on: Postadresse, Besøksadresse and Forretningsadresse.

**Comment by olemartinorg** [comment]
@hflateland We're spending most of our time tying up the loose ends in the next major release, v4, at the moment. I think this change should be an easy fix, and it might be a good thing to bring into v4 as it's a breaking change for all those apps who've already set a `title` binding.

**Comment by Magnusrm** [comment]
Fixed by @adamhaeger in #1880 🎉
